### PR TITLE
Update integrations-3.md

### PR DESCRIPTION
**Pull request by kengie**
updated tutorial for Launchpad, changes to wording and framing.

### CODE REVIEW

File: _guides/integrations-3.md [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+* Running integrations on OpenLattice servers can be faster by orders of magnitude.
 
-Your data would be stored in a private database accessible only to you - **please contact <a href="mailto:support@topenlattice.com">support@openlattice.com</a> for arrangements.** Later, one can still write your own integration script if desired and also put it onto OpenLattice servers to run automatically. A tutorial for this final step is being developed. 
+## 2. Launchpad
+Data transfers are obviously integral to using the OpenLattice platform, and we take very seriously the need to provide the highest level of security and transparency. For these reasons we have developed a secure and completely open access solution.
 
-Some advantages of this workflow include:
+Our [Launchpad](https://github.com/openlattice/launchpad) application eliminates the need for, and questionable security of, email file transfers by allowing you to transfer whole tables or subsets of tables, from your computer or SQL server directly to an OpenLattice server for later integration. Some advantages of this workflow include:
```
Comment (katherinebernstein) [inline _guides/integrations-3.md]: Maybe "eliminates the need for, and insecurity of, ..." since we're not eliminating security

File: _guides/integrations-3.md [hunk 2]
```diff
@@ -1,30 +1,35 @@
 ---
 layout: page
-title: Integrations Part 3 – Sending Data directly to OpenLattice's Servers
-description: Bypass manual transfer of .csv files by connecting your database directly to OpenLattice. This guide is recommended for individuals who are interested in setting up integrations into the OpenLattice platform where data is transferred directly and securely from database to database. Reach out to support@openlattice.com if you need any assistance with your integration.
+title: Integration Pipeline: Secure Data Transfers & Speed
+description: Maximize both security and performance by using our **Launchpad** application to transfer data directly and securely from your database directly into OpenLattice. This is the recommended data pipeline for using the OpenLattice platform. Reach out to support@openlattice.com if you need any assistance with your integration.
```
Comment (UnsungHero97) [inline _guides/integrations-3.md]: `directly and securely [...] directly into`, using "directly" twice is redundant. Can you rephrase? Perhaps `... to transfer data from your database directly and securely into OpenLattice`?

File: _guides/integrations-3.md [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
+To give an overview of the process, a temporary table is copied over from a client-side server into OpenLattice's "Atlas" production server, which is set up on [AWS GovCloud](https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/whatis.html).  OpenLattice then runs all data integrations on our side, on these temporary tables. For those that want continuous integrations (for instance, to add to and update CAD police data or behavioral health reports as they come in), the data transfer can be set up as a recurring task on the client-side computer to run at a set time period and over-write the table that exists on the OpenLattice server, which is then integrated in parallel. These overwrites also ensure that the tables copied onto the OpenLattice side are temporary.
+
+Your data would be stored in a private database accessible only to you - **please contact <a href="mailto:support@topenlattice.com">support@openlattice.com</a> for arrangements.** Later, one can still write your own integration script if desired and also put it onto OpenLattice servers to run automatically. A tutorial for this final step is being developed.
```
Comment (katherinebernstein) [inline _guides/integrations-3.md]: Super minor but should probably switch to either "Later, one can still write their own..." or "Later, you can still write your own..."

**Comment by kengie** [comment]
thanks!! Made suggested changes.
